ui/components/app/token-detected/token-address/token-address.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import TokenAddress from './token-address';
+
+export default {
+  title: 'Components/APP/token-detected/TokenAddress',
```

Comment:
Convention is to use camel case for the title here
```suggestion
  title: 'Components/App/TokenDetected/TokenAddress',
```